pegasus/sites.v3/code.org/views/theme_common_head_before.haml

```diff
@@ -18,7 +18,7 @@
     is_ie8 = true;
 
 -if use_min_stylesheet?
-  %link{rel:'stylesheet', type:'text/css', href:'/style-min.css'}
+  =inline_css 'style-min.css'
 -else
   %link{rel:'stylesheet', type:'text/css', href:'/style.css'}
```

Comment:
we should still use style.css if we're on csedweek.org too.  I'll prep a PR to fix.